### DISCUSSION TITLE
Groups access on version 3.2

### DISCUSSION THREAD

**Discussion by kamal-meeheelaul** [papermerge/papermerge-core#353] [General]
I have just installed the version 3.2. I have created a group, into which I have added 3 users. The idea is, the 3 users are in the same department, and if User-1 creates a folder and uploads a document, User-2 should be able to access the document. However, this is not the case. Is it by design? Or a bug?

**Comment by ciur** [comment]
It is not the case yet, because  "share documents/folders feature between users" feature is not there yet.
At this point groups serve for limiting user access (authentication):  [example here]( https://docs.papermerge.io/3.2/sso/oidc/keycloak/#groups-permissions)

The "share documents/folders feature between users" feature is coming soon.

**Reply by kamal-meeheelaul** [reply]
Thanks for sharing! Looking forward to this feature.
